src/realsense.def

```diff
@@ -263,6 +263,7 @@ EXPORTS
     rs2_serialize_json
     rs2_emitter_frequency_mode_to_string
     rs2_depth_auto_exposure_mode_to_string
+    rs2_gyro_sensitivity_to_string
```

Comment:
Needed? we don't have a new API

Reply:
@Nir-Az is it needed? we are not sure also the lines about emitter and auto_exposure.

Reply:
We did.
in `rs-option.h`
![image](https://github.com/IntelRealSense/librealsense/assets/64067618/1549c9e4-c26e-475d-a585-ece52aae7e2f)

Reply:
No change here